### ISSUE TITLE
Kriminologie: Mehrere Exemplardaten bei einem Datensatz

### ISSUE THREAD

**Issue by IxKo** [ubtue/DatenProbleme#1944]
#### URL
https://doi.org/10.15845/bjclcj.v10i1.3686

Ein Extrem-Beispiel gemeldet von IxTheoKm.
PPN: 180181743X
Hashs/Exemplarsätze an diesem Datensatz:
KrimDok#2022-05-13#081DFA2BB087F1ADEF66BFA5DEAF2CA60D6EBCDD
KrimDok#2022-07-08#081DFA2BB087F1ADEF66BFA5DEAF2CA60D6EBCDD
KrimDok#2022-07-26#081DFA2BB087F1ADEF66BFA5DEAF2CA60D6EBCDD
KrimDok#2022-08-28#081DFA2BB087F1ADEF66BFA5DEAF2CA60D6EBCDD
KrimDok#2022-08-31#081DFA2BB087F1ADEF66BFA5DEAF2CA60D6EBCDD
KrimDok#2022-10-02#9905CCEAE5E67E50CE9735D8A7801A857D798D4D
KrimDok#2022-10-24#9905CCEAE5E67E50CE9735D8A7801A857D798D4D
KrimDok#2022-10-27#E6ABC8DB89006B55C9166204FB297C25B02C8F38
KrimDok#2022-11-07#E6ABC8DB89006B55C9166204FB297C25B02C8F38
KrimDok#2022-11-20#E6ABC8DB89006B55C9166204FB297C25B02C8F38
KrimDok#2022-11-30#E6ABC8DB89006B55C9166204FB297C25B02C8F38
KrimDok#2022-12-29#E6ABC8DB89006B55C9166204FB297C25B02C8F38
  
Es existieren laut IxTheoKm noch weitere solche Fälle.

Es fanden wohl sehr viele Umlenkungen (11 Stück, Feld 2119) statt, wodurch wahrscheinlich auch die Links doppelt aufgeführt werden:

![image](https://user-images.githubusercontent.com/57000004/232528224-ecc022bb-d0c9-4281-ae9d-f603a44d2793.png)

Die Fragen sind:
Wie ist das passiert?
Wie kann es zukünftig verhindert werden?

**Comment by jriedl** [comment]
Die Ursache für den Mehrfachupload ist vermutlich die gleiche wie die im Zusammenhang mit #1907 aufgefallene Botproblematik (vgl. Mail vom 15.2. 14:28). Dies sollte zukünftig nicht mehr auftreten. Überraschend erscheint mir allerdings, dass dies nicht wie im ersten Fall vollständing über das M&M abgefangen wurde und wie mit den Einspielungen umgegangen wurde: PPNs 1819843092,1821140109,1830127977,1824115083 sind z.B. als eigene Datensätze ohne Lokaldaten im letzten Komplettabzug enthalten, die zugehörigen Lokaldaten hängen aber scheinbar alle an 180181743X. Ist das eine zulässige Konstellation?

**Comment by IxKo** [comment]
> PPNs 1819843092,1821140109,1830127977,1824115083 sind z.B. als eigene Datensätze ohne Lokaldaten im letzten Komplettabzug enthalten, die zugehörigen Lokaldaten hängen aber scheinbar alle an 180181743X. Ist das eine zulässige Konstellation?

Bezieht sich das auf den VuFind?

Im K10plus gibt es nur noch die PPN 180181743X und alle anderen PPNs wurden zu dieser umgelenkt.

Demnach sollte das Problem in KrimDok mit einem neuen Abzug gelöst sein? Hier kenne ich mich nicht aus.

**Comment by jriedl** [comment]
Ja, bezieht sich auf den aktuellen Krimdok-Datenbestand (vgl. "Weitere Versionen" in der Volltitelanzeige von 180181743X). Umlenkung bedeutet vermutlich, dass solche die PPNs nicht in die Löschlisten wandern, weil sie noch im K10Plus existieren. 180181743X hat als letztes Änderungsdatum den 9.1.23 (vgl. MARC 005). Unseren Komplettabzug haben wir am 11. Februar erhalten, d.h. falls die Umleitung eine Anpassung von 005 zur Folge hat, scheint ein neuer Komplettabzug das Problem nicht zu lösen. Falls eine manuelle Löschung in solchen Fällen nicht wünschenswert ist, müssten wir evtl. beim BSZ nachfragen, ob zukünftig bei einem Komplettabzug die umgeleiteten Records aus dem Abzug ausgeschlossen werden könnten, oder gibt es Fälle, in denen es sinnvoll erscheint, diese zu "behalten"?

**Comment by IxKo** [comment]
Im K10plus gibt es nur noch einen Datensatz, der aber - zur Nachvollziehbarkeit - alle PPNs (also nur die Nummern), die mit dem Datensatz zusammengeführt wurden, enthält. Diese PPNs stehen in den Feldern 2119 und 2241

**Comment by IxTheoKm** [comment]
Diese Art Umlenkungen/Dublettenbereinigungen gibt es schon seit langem - auch bei IxTheo/RelBib.
Irgendwie werden die bisher schon in den VuFind-Datenbanken nachvollzogen. Wie das technisch realisiert wird, weiß ich leider nicht.

**Comment by jriedl** [comment]
Anfrage ans BSZ läuft.
